### ISSUE TITLE
TextField labels shouldn't be constrained

### ISSUE THREAD

**Issue by AntonyKwok** [zopaUK/react-components#63]
Currently the label for a `TextField` will be constrained within the bounds of the wrapping div element. This behaviour prevents consumers to create a small `TextField` with a helpful label, for instance (with the current implementation):
![Screenshot 2019-08-09 at 16 15 29](https://user-images.githubusercontent.com/3268559/62789752-86879e00-bac1-11e9-89eb-12a02d63d419.png)

And here's what I suggest it should look like:
![Screenshot 2019-08-09 at 16 15 45](https://user-images.githubusercontent.com/3268559/62789829-ab7c1100-bac1-11e9-8495-09b8c11572ef.png)

**Comment by thegrinder** [comment]
@AntonyKwok thanks for creating the issue 👍 I just looked at the `<TextField />` and it seems that neither the label nor the error message are constrained 

<img width="516" alt="Screenshot 2019-08-09 at 17 51 44" src="https://user-images.githubusercontent.com/14250944/62791987-92c62800-bace-11e9-8482-644c755e5086.png">

Are you using the latest version? Cause I think there have been some recent changes related to this issue

**Comment by thegrinder** [comment]
yup, this issue has been solved https://github.com/zopaUK/react-components/pull/45
